### PR TITLE
Added the output for the provided cmdlet example

### PR DESCRIPTION
**Pull request by msewaweru**

### CODE REVIEW

**Comment by billmath** [comment]
Docs Build status updates of commit _[8fbee3b](https://github.com/msewaweru/azure-docs-powershell-azuread/commits/8fbee3b6fc25f946ea4b9fdac673ea4bc9435d66)_: 

### :x: Validation status: errors

Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=main#build--publish) which may help to resolve issue.


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[azureadps-2.0/AzureAD/New-AzureADMSInvitation.md](https://github.com/msewaweru/azure-docs-powershell-azuread/blob/issue-689/azureadps-2.0/AzureAD/New-AzureADMSInvitation.md) | :x:Error | | [Details](#user-content-3717c318118a3bb14f3793917dd6c351-88e88173)
[azureadps-2.0/AzureAD/index.yml](https://github.com/msewaweru/azure-docs-powershell-azuread/blob/issue-689/azureadps-2.0/AzureAD/index.yml) | :warning:Warning | | [Details](#user-content-37bfae49af1a8a6b2e01cf868c49bb89-88e88173)

<a id="3717c318118a3bb14f3793917dd6c351-88e88173"></a>
### [azureadps-2.0/AzureAD/New-AzureADMSInvitation.md](https://github.com/msewaweru/azure-docs-powershell-azuread/blob/issue-689/azureadps-2.0/AzureAD/New-AzureADMSInvitation.md)
 - **Line 0, Column 0**: **[Error: MAML2Yaml_LoadFileFailed]** 
`````
Failed to load file: W:/2yvk-s/azureadps-2.0/AzureAD/New-AzureADMSInvitation.md:128:(567) '[### System.Object](/graph/api/invitation-post?vie...'
 Expect Heading, please try to use PlatyPS validate the content
`````

<a id="37bfae49af1a8a6b2e01cf868c49bb89-88e88173"></a>
### [azureadps-2.0/AzureAD/index.yml](https://github.com/msewaweru/azure-docs-powershell-azuread/blob/issue-689/azureadps-2.0/AzureAD/index.yml)
 - **Line 167, Column 5**: **[Warning: xref-not-found]** `````Cross reference not found: 'AzureAD.Remove-AzureADMSRoleAssignment'.`````
 - **Line 168, Column 5**: **[Warning: xref-not-found]** `````Cross reference not found: 'AzureAD.Remove-AzureADMSRoleDefinition'.`````
 - **Line 169, Column 5**: **[Warning: xref-not-found]** `````Cross reference not found: 'AzureAD.Set-AzureADMSRoleDefinition'.`````
 - **Line 218, Column 5**: **[Warning: xref-not-found]** `````Cross reference not found: 'AzureAD.Set-AzureADMSAuthorizationPolicy'.`````

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2022%5C2%5C2%5Ced7d034d-a56c-a713-9d51-730d2ba643e8%5CPullRequest%5C202202021129320760-771%5Cworkflow_report.html?sv=2020-08-04&se=2022-03-05T11%3A31%3A03Z&sr=b&sp=r&sig=9H9FBQipCb48M%2BSgj3PiQh%2BqYsAgrKw217%2Bh4gUaI4o%3D).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=main) to re-authorize your GitHub account to Docs Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/ed7d034d-a56c-a713-9d51-730d2ba643e8?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=main&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by billmath** [comment]
Docs Build status updates of commit _[2b109d1](https://github.com/msewaweru/azure-docs-powershell-azuread/commits/2b109d184c3e10450279dca60f9002014760aca1)_: 

### :x: Validation status: errors

Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=main#build--publish) which may help to resolve issue.


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[azureadps-2.0/AzureAD/New-AzureADMSInvitation.md](https://github.com/msewaweru/azure-docs-powershell-azuread/blob/issue-689/azureadps-2.0/AzureAD/New-AzureADMSInvitation.md) | :x:Error | | [Details](#user-content-3717c318118a3bb14f3793917dd6c351-efb4e064)
[azureadps-2.0/AzureAD/index.yml](https://github.com/msewaweru/azure-docs-powershell-azuread/blob/issue-689/azureadps-2.0/AzureAD/index.yml) | :warning:Warning | | [Details](#user-content-37bfae49af1a8a6b2e01cf868c49bb89-efb4e064)

<a id="3717c318118a3bb14f3793917dd6c351-efb4e064"></a>
### [azureadps-2.0/AzureAD/New-AzureADMSInvitation.md](https://github.com/msewaweru/azure-docs-powershell-azuread/blob/issue-689/azureadps-2.0/AzureAD/New-AzureADMSInvitation.md)
 - **Line 0, Column 0**: **[Error: MAML2Yaml_LoadFileFailed]** 
`````
Failed to load file: W:/cbl3-s/azureadps-2.0/AzureAD/New-AzureADMSInvitation.md:128:(567) '[System.Object](/graph/api/invitation-post?view=gr...'
 Expect Heading, please try to use PlatyPS validate the content
`````

<a id="37bfae49af1a8a6b2e01cf868c49bb89-efb4e064"></a>
### [azureadps-2.0/AzureAD/index.yml](https://github.com/msewaweru/azure-docs-powershell-azuread/blob/issue-689/azureadps-2.0/AzureAD/index.yml)
 - **Line 167, Column 5**: **[Warning: xref-not-found]** `````Cross reference not found: 'AzureAD.Remove-AzureADMSRoleAssignment'.`````
 - **Line 168, Column 5**: **[Warning: xref-not-found]** `````Cross reference not found: 'AzureAD.Remove-AzureADMSRoleDefinition'.`````
 - **Line 169, Column 5**: **[Warning: xref-not-found]** `````Cross reference not found: 'AzureAD.Set-AzureADMSRoleDefinition'.`````
 - **Line 218, Column 5**: **[Warning: xref-not-found]** `````Cross reference not found: 'AzureAD.Set-AzureADMSAuthorizationPolicy'.`````

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2022%5C2%5C2%5Ced7d034d-a56c-a713-9d51-730d2ba643e8%5CPullRequest%5C202202021132292460-771%5Cworkflow_report.html?sv=2020-08-04&se=2022-03-05T11%3A33%3A22Z&sr=b&sp=r&sig=vt8ggm5pccAmNREMbpO6Va5EnNgCVhPe%2Bz4bBDgZpl4%3D).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=main) to re-authorize your GitHub account to Docs Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/ed7d034d-a56c-a713-9d51-730d2ba643e8?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=main&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by billmath** [comment]
Docs Build status updates of commit _[0d347f6](https://github.com/msewaweru/azure-docs-powershell-azuread/commits/0d347f612a3c52c6c1e6dffadbb377393684236d)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[azureadps-2.0/AzureAD/New-AzureADMSInvitation.md](https://github.com/msewaweru/azure-docs-powershell-azuread/blob/issue-689/azureadps-2.0/AzureAD/New-AzureADMSInvitation.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/azuread/new-azureadmsinvitation?view=azureadps-2.0&branch=pr-en-us-771) (azureadps-2.0) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2022%5C2%5C2%5Ced7d034d-a56c-a713-9d51-730d2ba643e8%5CPullRequest%5C202202021146569824-771%5Cworkflow_report.html?sv=2020-08-04&se=2022-03-05T11%3A48%3A13Z&sr=b&sp=r&sig=40cmlrag12pOcB7MmcTCew92Guwfp5D006lFfD5F%2B5g%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/ed7d034d-a56c-a713-9d51-730d2ba643e8?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by billmath** [comment]
Docs Build status updates of commit _[a15d9a3](https://github.com/msewaweru/azure-docs-powershell-azuread/commits/a15d9a36e0f1a81fca9711ea13a28a14aeaf4815)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[azureadps-2.0/AzureAD/New-AzureADMSInvitation.md](https://github.com/msewaweru/azure-docs-powershell-azuread/blob/issue-689/azureadps-2.0/AzureAD/New-AzureADMSInvitation.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/azuread/new-azureadmsinvitation?view=azureadps-2.0&branch=pr-en-us-771) (azureadps-2.0) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2022%5C2%5C2%5Ced7d034d-a56c-a713-9d51-730d2ba643e8%5CPullRequest%5C202202021151199045-771%5Cworkflow_report.html?sv=2020-08-04&se=2022-03-05T11%3A52%3A29Z&sr=b&sp=r&sig=XUFEh1y1t%2BUV5PLO6T%2BOetJpeGE7cJ%2FPcQihsB7NLLQ%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/ed7d034d-a56c-a713-9d51-730d2ba643e8?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by msewaweru** [comment]
Fixes #689

**Review by jpettere — APPROVED** [review]

**Comment by msewaweru** [comment]
@CelesteDG , please review this.

File: azureadps-2.0/AzureAD/New-AzureADMSInvitation.md [hunk 1]
```diff
@@ -26,6 +26,26 @@ This cmdlet is used to invite a new external user to your directory.
...
+                            CcRecipients: System.Collections.Generic.List`1[Microsoft.Open.MSGraph.Model.Recipient]
+                            CustomizedMessageBody:
+                            MessageLanguage:
+                          }
+
+InvitedUserType         : Guest
+Status                  : PendingAcceptance
 ```
 
 Using the cmdlet in this example, an email is sent to the user who's email address is in the -InvitedUserEmailAddress parameter.
```
Comment (CelesteDG) [inline azureadps-2.0/AzureAD/New-AzureADMSInvitation.md:51]:
```suggestion
Using the cmdlet in this example, an email is sent to the user whose email address is in the -InvitedUserEmailAddress parameter.
```

**Review by CelesteDG — CHANGES_REQUESTED** [review]

**Comment by CelesteDG** [comment]
Please note the difference between "who's" and "whose"

**Comment by billmath** [comment]
Docs Build status updates of commit _[31c29f7](https://github.com/msewaweru/azure-docs-powershell-azuread/commits/31c29f7828fa5db7e1b10c9c49e6175bd910f7ed)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[azureadps-2.0/AzureAD/New-AzureADMSInvitation.md](https://github.com/msewaweru/azure-docs-powershell-azuread/blob/issue-689/azureadps-2.0/AzureAD/New-AzureADMSInvitation.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/azuread/new-azureadmsinvitation?view=azureadps-2.0&branch=pr-en-us-771) (azureadps-2.0) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2022%5C2%5C3%5Ced7d034d-a56c-a713-9d51-730d2ba643e8%5CPullRequest%5C202202030748554271-771%5Cworkflow_report.html?sv=2020-08-04&se=2022-03-06T07%3A49%3A54Z&sr=b&sp=r&sig=kS404qMhVeJr0dk9yQQKxWXhRoha%2BETHXivqc%2FDki2g%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/ed7d034d-a56c-a713-9d51-730d2ba643e8?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by billmath** [comment]
Docs Build status updates of commit _[31c29f7](https://github.com/msewaweru/azure-docs-powershell-azuread/commits/31c29f7828fa5db7e1b10c9c49e6175bd910f7ed)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[azureadps-2.0/AzureAD/New-AzureADMSInvitation.md](https://github.com/msewaweru/azure-docs-powershell-azuread/blob/issue-689/azureadps-2.0/AzureAD/New-AzureADMSInvitation.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/azuread/new-azureadmsinvitation?view=azureadps-2.0&branch=pr-en-us-771) (azureadps-2.0) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2022%5C2%5C3%5Ced7d034d-a56c-a713-9d51-730d2ba643e8%5CPullRequest%5C202202030749057775-771%5Cworkflow_report.html?sv=2020-08-04&se=2022-03-06T07%3A51%3A12Z&sr=b&sp=r&sig=jNwfyX3aOvjIGYGFuSgmSWeaaVN9JX0b7nnHS%2Br%2BvlM%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/ed7d034d-a56c-a713-9d51-730d2ba643e8?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by msewaweru** [comment]
@CelesteDG, this is good to go.

**Review by CelesteDG — APPROVED** [review]

**Pull request merged**